common/changes/@itwin/ecschema-editing/Frank-updateEnumsInCore-quantity_2024-10-09-13-45.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/ecschema-editing",
+      "comment": "refactored the toString methods used in this file",
```

Comment:
this isn't something users will see or notice, so you can leave the comment as an empty string. Same with the changelog file in ecschema-metadata